elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -1422,6 +1423,16 @@ public void addArgumentsToAttributes(Class<?> cls, String attributeName, Set<Arg
         getEntityBinding(cls).addArgumentsToAttribute(attributeName, arguments);
     }
 
+    /**
+     * Add a single argument to the attribute
+     * @param cls The entity
+     * @param attributeName attribute name to which argument has to be added
+     * @param argument A single argument
+     */
+    public void addArgumentToAttributes(Class<?> cls, String attributeName, ArgumentType argument) {
```

Comment:
It looks like this is `addArgumentToAttribute`, no `s` at the end.

Reply:
👍